pom.xml

```diff
@@ -80,11 +80,11 @@
         <plugin>
           <artifactId>maven-surefire-plugin</artifactId>
           <version>3.2.5</version>
-           <configuration>
+           <!-- <configuration>
```

Comment:
**Issue**: Commenting out the <configuration> tags without explanation may confuse future contributors. <br> **Fix**: Provide a reason for commenting out these tags or remove them if they are unnecessary. <br> **Code Suggestion**: 
 ```
 -           <!-- <configuration>
 +           <configuration>
              <suiteXmlFiles>
                    <suiteXmlFile>src/test/resources/testng.xml</suiteXmlFile>
              </suiteXmlFiles>
 -      </configuration> -->
 +      </configuration>
 ```